dapr/clients/grpc/_state.py

```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+from dapr.proto import common_v1
+
+
+class State:
+    def __init__(self, key, value, etag, options: StateOptions = None):
+        self.key = key
+        self.value = value
+        self.etag = etag
+        self.options = options
+
+
+class Consistency(Enum):
+    eventual = common_v1.StateOptions.StateConsistency.CONSISTENCY_EVENTUAL
+    strong = common_v1.StateOptions.StateConsistency.CONSISTENCY_STRONG
+
+
+class Concurrency(Enum):
+    first_write = common_v1.StateOptions.StateConcurrency.CONCURRENCY_FIRST_WRITE
+    last_write = common_v1.StateOptions.StateConcurrency.CONCURRENCY_LAST_WRITE
+
+
+class RetryPolicy:
+    def __init__(self, threshold, interval, pattern: RetryPattern):
+        self.threshold = threshold
+        self.interval = interval
+        if pattern is None:
+            self.pattern = common_v1.StateOptions.RetryPattern.RETRY_UNSPECIFIED
+
+
+class RetryPattern(Enum):
+    linear = common_v1.StateOptions.StateRetryPolicy.RetryPattern.RETRY_LINEAR
+    exponential = common_v1.StateOptions.StateRetryPolicy.RetryPattern.RETRY_EXPONENTIAL
```

Comment:
both retry related protobuf msgs will be deprecated in this milestone. please move RetryPolicy and retryPattern.

Reply:
`dapr/clients/grpc/_state.py:14: error: "GeneratedProtocolMessageType" has no attribute "StateConsistency"  [attr-defined]` 
I keep getting type errors when I try to import them. Is there a way to resolve it?
```
class Consistency(Enum):
    eventual = common_v1.StateOptions.StateConsistency.CONSISTENCY_EVENTUAL
    strong = common_v1.StateOptions.StateConsistency.CONSISTENCY_STRONG
```

Reply:
please ignore typing error by adding `# type: ignore` at the end of line.